lib/vertica/row.rb

```diff
@@ -13,23 +15,44 @@ def fetch(name_or_index)
     @values.fetch(column_index(name_or_index))
   end
 
-  def inspect
-    "<Vertica::Row#{@values.inspect}>"
-  end
-
   alias_method :[], :fetch
 
+  # Returns an array representation of the row. The values will
+  # be ordered like the order of the fields in the {#row_description}.
+  # @return [Array]
   def to_a
-    @values.to_a
```

Comment:
I use `RowDescription#build_row` to make sure a row is always initialized with an array.